system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -86,19 +90,22 @@ impl pallet_referenda::Config<FellowshipReferendaInstance> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type Scheduler = Scheduler;
 	type Currency = Balances;
-	// Fellows can submit proposals.
+	// Fellows & Secretary can submit proposals.
 	type SubmitOrigin = EitherOf<
-		pallet_ranked_collective::EnsureMember<Runtime, FellowshipCollectiveInstance, 3>,
-		MapSuccess<
-			TryWithMorphedArg<
-				RuntimeOrigin,
-				<RuntimeOrigin as OriginTrait>::PalletsOrigin,
-				ToVoice,
-				EnsureOfRank<Runtime, FellowshipCollectiveInstance>,
-				(AccountId, u16),
+		EitherOf<
+			pallet_ranked_collective::EnsureMember<Runtime, FellowshipCollectiveInstance, 3>,
```

Comment:
While I know that I said this. The actual problem here is that we are missing a proper UI and don't really need this. Can you revert?

Reply:
> Can you revert?

Certainly! 

> The actual problem here is that we are missing a proper UI

What would a proper UI need to enable this feature? Usually thought the secretary signature would allow submission on the main UI.